.golangci.yaml

```diff
@@ -1,5 +1,7 @@
 run:
   timeout: 5m
+  skip-files:
+    - pkg/target/matchcrd_constant.go
```

Comment:
I think you also need to skip this file in the `lint` target in the makefile.

Reply:
I believe the [`lint` target](https://github.com/open-policy-agent/gatekeeper/blob/master/Makefile#L273) runs this golangci-lint, so the file exclusion should be covered across CI and local development. 

To double check, I added some weird formatting to `matchcrd_constant.go` and ran `make lint`. No errors reported.